cpp/tests/community/triangle_count_test.cpp

```diff
@@ -278,7 +290,10 @@ INSTANTIATE_TEST_SUITE_P(
   Tests_TriangleCount_Rmat,
   ::testing::Combine(
     // enable correctness checks
-    ::testing::Values(TriangleCount_Usecase{0.1}, TriangleCount_Usecase{1.0}),
+    ::testing::Values(TriangleCount_Usecase{0.1, false},
+                      TriangleCount_Usecase{0.1, true},
+                      TriangleCount_Usecase{1.0, false},
+                      TriangleCount_Usecase{1.0, true}),
```

Comment:
Why correctness checks are not either all false or all true?

Reply:
See
```
struct TriangleCount_Usecase {
  double vertex_subset_ratio{0.0};

  bool edge_masking{false};
  bool check_correctness{true};
};
```
This has 3 member variables, here I am providing only two, so the third one (check_correctness) is set to the default value (which is false).

Reply:
Ah okay.